gtsfm/utils/metrics.py

```diff
@@ -422,3 +422,19 @@ def save_metrics_as_json(metrics_groups: Delayed, output_dir: str) -> None:
     """
     for metrics_group in metrics_groups:
         metrics_group.save_to_json(os.path.join(output_dir, metrics_group.name + ".json"))
+
+
+def get_stats_for_sfmdata(gtsfm_data: GtsfmData, suffix: str) -> List[GtsfmMetric]:
```

Comment:
moved this function from scene_optimizer

Reply:
Is it only used in this file?

Reply:
Yep. But I am moving it because it is not a core scene-optimizer functionality (which is to call different modules)